starwars/src/components/CharacterCards.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "styled-components";
+
+const Card = styled.div`
+    padding: 50px;
+`;
+
+const CardBody = styled.div`
+    background: moccasin;
+    padding: 20px;
+    border-radius: 10px
+`;
+
+const CardTitle = styled.h1`
+    font-size: 25px;
+`;
+
+const CardSubtitle = styled.h3`
+    font-size: 15px;
+`;
+
+const CharacterCards = props => {
+    return (
+        <Card>
+          <CardBody>
+            <CardTitle>{props.name}</CardTitle>
+            <CardSubtitle>Birth Year: {props.birthYear}</CardSubtitle>
+            <CardSubtitle>Eye Color: {props.eyeColor}</CardSubtitle>
+            <CardSubtitle>Hair Color: {props.hairColor}</CardSubtitle>
+            <CardSubtitle>Height: {props.height}</CardSubtitle>
+            <CardSubtitle>Mass: {props.mass}</CardSubtitle>
```

Comment:
Try to use more semantic elements here instead of multiple CarSubtitle components. Maybe use a CardText component and inside it use a ul. Something like this:
`
<CardText>
    <ul>
        <li>Birth Year: { props.birthyear }</li>
    </ul>
</CardText>
`